src/exosite.c

```diff
@@ -376,8 +426,20 @@ exo_state exo_operate(exo_op *op, uint8_t count)
       break;
     }
   }
+}
+
+// process all ops that are in an active state
+static void exo_process_active_ops(exo_op *op, uint8_t count)
+{
+  uint8_t buf[576];
```

Comment:
I think forcing c99 may be too much.  We have a customer now whose compiler is only c89, so it is very possible we would have this limitation in the future.  Would it possible to have a flag compile one way or the other?